scripts/dev/gen_stub.php

```diff
@@ -455,6 +549,44 @@ function generateArginfoCode(array $funcInfos): string {
     return $code . "\n";
 }
 
+/** @param FuncInfo[] $funcInfos */
+function generateZppCode(array $funcInfos): string {
+    $code = "/* This is a generated file, edit the .stub.php file instead. */";
+    $generatedZppFuncInfo = [];
+    foreach ($funcInfos as $funcInfo) {
+        if ($funcInfo->hasCompleteParameterListTypeInfo() === false) {
+            continue;
+        }
+
+        $code .= "\n\n";
+        if ($funcInfo->cond) {
+            $code .= "#if {$funcInfo->cond}\n";
+        }
+
+        /* If there already is an equivalent ZPP macro, only emit a #define */
+        if (false && $generatedFuncInfo = findEquivalentFuncInfoByParameters($generatedZppFuncInfo, $funcInfo)) {
```

Comment:
Didn't get this condition

Reply:
The `false` is just put there in order to disable the `#define`s. They didn't yet work for me reliably due to `#ifdef`s so I just wanted to postpone the fix until the PR gets feedback.